extensions/default/src/getSopClassHandlerModule.js

```diff
@@ -61,7 +61,23 @@ function getDisplaySetInfo(instances) {
   };
 }
 
+const instanceSort = (a, b) => {
```

Comment:
we have couple of sort functions in here platform/core/src/utils/sortStudy.ts can you see if you can update them or use one of them instead of adding one here?